README.md

```diff
@@ -153,8 +153,8 @@ server {
 }
 ```
 
-Prior to using MySQL, MongoDB or SQLite you need to create the database and tables using [one of the provided files](db).  
```

Comment:
Yeah, I'll tidy it up at some point. Trailing whitespace to denote meaning. Wat.